routes/html-routes.js

```diff
@@ -17,6 +21,11 @@ module.exports = (app) => {
     // If the user already has an account send them to the members page
     if (req.user) {
       res.redirect('/members');
+      // if (req.user.user_type === 'client') {
```

Comment:
are we going to need the commented code later?

Reply:
no, i've made a new commit to remove this section